### DISCUSSION TITLE
error: path 'LAN_CLI_B' resolves to multiple networks

### DISCUSSION THREAD

**Discussion by dikobrazsd** [vmware/govmomi#3525] [Community Help] [answered]
While trying to clone VM, getting this error:

`path 'LAN_CLI_B' resolves to multiple networks.`

I am using -net "LAN_CLI_B".

LAN_CLI_B resolves to two different networks with the same name:

`# govc find . -type n |grep CLI_B`
`/DC1/network/LAN_CLI_B`
`/DC1/network/LAN_CLI_B`

One is distributed port group.

How do I clone a VM and select one network over another? Can I use an ID or something similar to identify a specific network?

VSphere search shows this:

Search for "LAN_CLI_B" 
(2 results , including 1  Network, 1  Distributed Port Group)

**Comment by dougm** [comment, marked as answer]
Using `SwitchName/PortgroupName` is an option if that's unique in your env. Example here, but applies to any command with `-net` flag: https://github.com/vmware/govmomi/blob/main/govc/USAGE.md#vmnetworkadd

A few more options in the doc here: https://github.com/vmware/govmomi/blob/c7370668947f5f74b0bd03a1e1d0980583220cb7/find/finder.go#L798-L809

The tree command also shows `SwitchName/PortgroupName`:
```console
% govc tree -p network
network
├── [Network] VM Network
├── [VmwareDistributedVirtualSwitch] network-1-vds
│   ├── [DistributedVirtualPortgroup] network-1
│   └── [DistributedVirtualPortgroup] network-1-vds-DVUplinks-60
└── [VmwareDistributedVirtualSwitch] primary-vds
    ├── [DistributedVirtualPortgroup] primary
    └── [DistributedVirtualPortgroup] primary-vds-DVUplinks-54
```
